frontend/src/HomePage/AppCard.jsx

```diff
@@ -90,7 +110,9 @@ export default function AppCard({
                   canUpdateApp={canUpdateApp(app)}
                   deleteApp={() => deleteApp(app)}
                   exportApp={() => exportApp(app)}
-                  isMenuOpen={isMenuOpen}
+                  isMenuOpen={setMenuOpen}
```

Comment:
Hey, @kushalsourav why was this changed to setMenuOpen

Reply:
Hey @rudeUltra  thank you for noticing this. I forgot to remove that line. The `isMenuOpen` prop is not defined as a prop in  `AppMenu.jsx`. I think we don’t need that line

* The issue was to fix the screen breaking at the bottom edge when the last row's app menu popup is visible
* I got an update on that issue: when the user clicks the button to open the pop-up, scrolling past the `App Card` linked to the pop-up closes it
* After fixing that, I realized the Home page has pagination, as users won’t be scrolling down when the screen width is at its maximum
* Popups auto close works only when screen-Y (Y-axis) is scrollable